Xamarin.Forms.Platform.iOS/ViewRenderer.cs

```diff
@@ -18,11 +18,16 @@ namespace Xamarin.Forms.Platform.iOS
 namespace Xamarin.Forms.Platform.MacOS
 #endif
 {
+	public interface INativeViewRenderer
```

Comment:
`public` -> `internal`

Reply:
I would really prefer that you don't do this. I realise that you're trying to reduce your maintenance surface but at the same time you are cutting off at the knees anyone who wants to use this interface in their own code. If it's at all possible to _not_ make things `internal` please please don't do it. 
I once had to re-implement all of the `WebViewRenderer` functionality in a custom renderer to get around the early rotation-handling issues in the original. Because everything was `internal`, to fix a fundamental bug I had to reproduce almost the entire renderer in my code rather than simply adjusting a small piece of it.
IMO _all_ the methods of a renderer should be `public` and `virtual` or `override`. When we're creating renderers we simply don't know enough of the use cases to lock anything down.

Reply:
Well, you're not wrong that leaving it internal would keep custom renderers from being tab stops.

The current interface name (`INativeViewRenderer`) doesn't express what it does (and why it exists) very well; the classes implementing it actually have several 'native' views. What this is _really_ saying is "of all the native  views that make up this renderer, here's the one that makes sense as a tab stop".

So a better name for the interface would be `ITabStop`, with a single `TabStop` property that returns a native view. 

Also, all of the fast renderers on Android are missing this interface right now (the fast Button, Image, etc.) because they don't derive from `ViewRenderer`. 

All of which leads to me wonder if we should just add a `TabStop` property to `IVisualElementRenderer` on Android and iOS.

@samhouts Any thoughts on this? Between `ITabStop` and just adding the property to `IVisualElementRenderer`, do you see a clearly better option? Or a third way that we're missing?

Reply:
Now that I think about it some more, we can't add it to `IVisualElementRenderer` without breaking existing implementations. So I think `ITabStop` is the way to go, and it will need to be applied to the fast renderers.

Reply:
I agree. Nice catch!